### DISCUSSION TITLE
Interactsh server deployment to AWS EC2 Instance with Terraform

### DISCUSSION THREAD

**Discussion by vay3t** [projectdiscovery/interactsh#197] [Q&A] [answered]
### Please describe your feature request:
Create script for auto deploy instances in AWS and others like a this project https://github.com/Leoid/AWSBurpCollaborator

### Describe the use case of this feature:
minimize implementation times, in addition to the fact that the margin of error would decrease for newer users.

**Comment by denniskniep** [comment, marked as answer]
Have already done something like that. 

**Interactsh deployment to AWS EC2 Instance with Terraform:**
https://github.com/denniskniep/interactsh_deploy

Furthermore nameserver IP at Go Daddy will be automatically refreshed on (re)deployment

**Reply by vay3t** [reply]
I find it would be just as useful as godaddy to add support for Cloudflare.

https://api.cloudflare.com/#getting-started-endpoints
